README.md

```diff
@@ -1,11 +1,94 @@
 # Nextflow pipeline for running the bioBakery
 
-by Kevin Bonham, PhD
+by Kevin Bonham, PhD 
 
-- `KneadData`
-- `MetaPhlAn`
-- `HUMAnN`
+[bioBakery](https://github.com/biobakery): software, documentation, and tutorials for microbial community profiling (created and mantained by the Huttenhower lab)
 
-## Setup
+- [`KneadData`](https://github.com/biobakery/kneaddata): a data quality-control pipeline that removes host genomic data within our metagenomic samples. Particularly, this pipeline uses a database containing a reference human genome so that all human DNA is removed from the samples. Link to more information here: (https://huttenhower.sph.harvard.edu/kneaddata/).
+- [`MetaPhlAn`](https://github.com/biobakery/MetaPhlAn): a computational tool for species-level microbial profiling (bacteria, archaea, eukaryotes, and viruses) from metagenomic shotgun sequencing data. Link to more information here:(https://huttenhower.sph.harvard.edu/metaphlan)
+- [`HUMAnN`](https://github.com/biobakery/humann): a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. Link to more information here:(https://huttenhower.sph.harvard.edu/humann)
 
-**TODO**
\ No newline at end of file
+## Environment setup
+Instructions for setting up a local environment to run the pipeline can be found on Danielle's notebook [here](https://github.com/BonhamLab/daniellepinto/blob/main/PeriodicMeetings/2025-06-17.md#danielles-personal-notes). 
+
+Computing environments on the Tufts HPC and AWS should already be set-up with apptainer environments.
```

Comment:
```suggestion
Computing environments on the Tufts HPC and AWS should already be set-up with container-based (docker, apptainer) or conda environments.
```

Reply:
At some point we should add how this is done. Fine to leave it out of the initial draft